layout/ut/plan.c

```diff
@@ -0,0 +1,309 @@
+/* -*- C -*- */
+/*
+ * Copyright (c) 2012-2020 Seagate Technology LLC and/or its Affiliates
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For any questions about this software or licensing,
+ * please email opensource@seagate.com or cortx-questions@seagate.com.
+ *
+ * Original author: Andriy Tkachuk <andriy.tkachuk@seagate.com>
+ * Original creation date: 27-Apr-2021
+ */
+
+
+#include "lib/errno.h"     /* ENOENT */
+#include "lib/vec.h"       /* m0_indexvec_alloc */
+
+#define M0_TRACE_SUBSYSTEM M0_TRACE_SUBSYS_LAYOUT
+#include "lib/trace.h"     /* M0_LOG */
```

Comment:
Usually, these two lines should be the first lines, before any other "#include<>"

Reply:
Fixed.